src/__tests__/server-smoke.test.ts

```diff
@@ -0,0 +1,29 @@
+import { startServer } from '../server';
+import type { Server } from 'http';
+import request from 'supertest';
+
+jest.mock("../server", () => {
+  const original = jest.requireActual("../server");
+  return {
+      ...original,
+      ebbs: jest.fn()
+  };
+});
+
+describe('Server Smoke Test', () => {
+let server: Server;
+
+  beforeAll(async () => {
+    server = await startServer(9080);
```

Comment:
nit: let the OS automatically choose a free port by specifying port 0, to prevent conflicts.

Reply:
Cool, thanks.  920ff511349f41f7b015fb7712f9b9d3f682e87f